apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V39__Table_and_column_rename.sql

```diff
@@ -0,0 +1,308 @@
+--
```

Comment:
should this script have a name similar to the other ones since it is for (primarily?) the Beneficiary table? I see there are some other bits and pieces related to other tables -- does it make sense to move those to the corresponding table script so that we truly have one script per table?

Reply:
I renamed this to (sort of) to match the other scripts; however, since this is the first script it handles hsql stuff like making sure there are no foreign key constraints by dropping them; need that sort of stuff here, 'cuz pretty much all other tables have a FK constraint vs. bene_id in the beneficiary table. Remember, by the time we get to this script (V39) the hsql db is in place and  has all the FK constraints, etc. so we need to clean that up before we can even begin.

Also, by definition, we don't have 1 script == 1 table; for once we really don't need that level of granularity and for the claims tables, it is just easier to see/review the claim line items table and parent table in one place. Also, allows us to then do (add) the FK constraints between line items and parent claim, and parent claim to beneficiary table needed for hsql. I suppose the alternative could be we just do ALL the hsql FK constraints in a completely seperate scriipt, but then you will have to cross-reference between multiple files.